Lombiq.ContentEditors.Samples/Models/AsyncEditorEmployeePart.cs

```diff
@@ -3,9 +3,10 @@
 
 namespace Lombiq.ContentEditors.Samples.Models;
 
-// This is the part that holds the content fields of an employee that you want to show in your async editor. Note that
-// using content fields is not required or specific to async editors. You can use properties and editor shapes that
-// will be placed on the async editor page (ie. Orchard Core editor group).
+// This is the part that holds the content fields of an employee that you want to show in your async editor. Note, that
+// using content parts is not required or specific to async editors. You can use properties and editor shapes that will
+// be placed on the async editor page (ie. Orchard Core editor group). For demonstration purposes we use content fields
+// for the Employee content type.
```

Comment:
Why aren't we using view models? Or can we? Add it to the comment.

Reply:
I'm not 100% sure what's missing from here at this point. I believe, it clearly states that we are doing everything with content fields. If we are going with custom properties, content drivers, clearly we can and we should use view models, but there's an example for that. I'll try to overexplain it, but I guess you should suggest me something because I'm really not sure what I'm missing from the explanation.

Reply:
I've rephrased it to this:
// This is the part that holds the content fields of an employee that you want to show in your async editor. For this
// demonstration we are storing the values in content fields so that we don't need to create a content driver and shapes
// with view models as those are already taken care of by Orchard Core. Alternatively, you can use properties and editor
// shapes with their own view models; see the other example for the front-end async editor to see how to do that.